### DISCUSSION TITLE
I see this is made specifically for the BlackBerry Q10 but I wonder if it would work on a Blackberry Priv ?

### DISCUSSION THREAD

**Discussion by MantleFounder** [Dakkaron/Fairberry#13] [Q&A]
As far as I am concerned the keyboard (layout + # of keys) is the exact same. 

Am I missing something ?

![image](https://github.com/Dakkaron/Fairberry/assets/38961332/07edfa73-606e-4e98-a054-d70874b20546)

**Comment by Dakkaron** [comment]
The problem is the keyboard. The connector and pinout of the keyboards change with each BB phone, and the Q10 is the only one where there's propper documentation on it.

Also, the Q10 keyboards are much cheaper and more readily available.

But if you have a few spare priv keyboards around and want to reverse engineer them, it would be pretty easy to adjust the code and PCB to work with a different keyboard.
